.gitignore

```diff
@@ -8,6 +8,7 @@ node_modules
 # binaries
 dist
 binary_dist
+sim/web
```

Comment:
We don't want to ignore the whole directory, just the file. I actually just added this change in my recent PR too. You can probably just remove this one :)